app/src/docs/components/Search/index.jsx

```diff
@@ -115,9 +114,9 @@ type Props = {
 }
 
 const UnstyledSearch = ({ toggleOverlay, nav, ...props }: Props) => {
-    const [index, store] = useGetIndexStore()
     const [query, setQuery] = useState('')
-    const searchResults = useLunr(query, index, store)
+
+    const searchResults = useLunr(query)
```

Comment:
@fonty1 check this ooot. `useLunr` has the index and the store baked into itself. No need to pass them along.